plugins/utilities.json

```diff
@@ -3,6 +3,101 @@
   "description": "Utilities",
   "plugins_base_url": "https://github.com/bombsquad-community/plugin-manager/{content_type}/{tag}/plugins/utilities",
   "plugins": {
+    "sandbox": {
```

Comment:
Please add something which tells the user that it works only in local games / HOSTS